builder/osc/common/block_device.go

```diff
@@ -34,87 +35,89 @@ type LaunchBlockDevices struct {
 	LaunchMappings []BlockDevice `mapstructure:"launch_block_device_mappings"`
 }
 
-func buildOscBlockDevicesImage(b []BlockDevice) []osc.BlockDeviceMappingImage {
-	var blockDevices []osc.BlockDeviceMappingImage
+func buildOscBlockDevicesImage(b []BlockDevice) []oscgo.BlockDeviceMappingImage {
+	var blockDevices []oscgo.BlockDeviceMappingImage
 
 	for _, blockDevice := range b {
-		mapping := osc.BlockDeviceMappingImage{
-			DeviceName: blockDevice.DeviceName,
+		mapping := oscgo.BlockDeviceMappingImage{
+			DeviceName: &blockDevice.DeviceName,
 		}
 
 		if blockDevice.VirtualName != "" {
 			if strings.HasPrefix(blockDevice.VirtualName, "ephemeral") {
-				mapping.VirtualDeviceName = blockDevice.VirtualName
+				mapping.VirtualDeviceName = &blockDevice.VirtualName
 			}
 		} else {
-			bsu := osc.BsuToCreate{
-				DeleteOnVmDeletion: blockDevice.DeleteOnVmDeletion,
+			bsu := oscgo.BsuToCreate{
+				DeleteOnVmDeletion: &blockDevice.DeleteOnVmDeletion,
 			}
 
 			if blockDevice.VolumeType != "" {
-				bsu.VolumeType = blockDevice.VolumeType
+				bsu.VolumeType = &blockDevice.VolumeType
 			}
 
 			if blockDevice.VolumeSize > 0 {
-				bsu.VolumeSize = int32(blockDevice.VolumeSize)
+				bsu.VolumeSize = aws.Int32(int32(blockDevice.VolumeSize))
```

Comment:
Use [this](https://github.com/outscale/osc-sdk-go/blob/7a11a0906c025139fb2b6f25cbd8f38890121333/v2/utils.go#L26) instead of AWS